python/Ganga/test/GPI/FileTests/TestMassStorage.py

```diff
@@ -0,0 +1,304 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+import tempfile
+import datetime
+import time
+import random
+import os
+import shutil
+import copy
+
+def generateUniqueTempFile( ext = '.txt' ):
+    """ Generate a unique file with a given filename with some random contents and return the name of the file on disk
+    Args:
+        ext (str): This is the extension (including '.') to give to the file of interest
+    """
+
+    myFile = tempfile.NamedTemporaryFile(mode='w',delete=False)
+
+    t = datetime.datetime.now()
+    unix_t = time.mktime(t.timetuple())
+
+    file_string = str(unix_t) + "\n"
+
+    random.seed( unix_t )
+    rand = random.random() * 1E10
+
+    file_string = file_string + str( rand ) + "\n"
+
+    urand = os.urandom(20)
+
+    file_string = file_string + str( urand ) + "\n"
+
+    myFile.write( file_string )
```

Comment:
I have been using this script elsewhere I just copied/pasted it here... I'll clean it up